tgui/packages/tgui/interfaces/PaperSheet/index.tsx

```diff
@@ -241,6 +236,15 @@ const PaperSheetEdit: React.FC<PaperSheetEditProps> = ({
   const [textAreaText, setTextAreaText] = useState('');
   const [combinedText, setCombinedText] = useState(value || '');
   const [showingHelpTip, setShowingHelpTip] = useState(false);
+  const [remarkable] = useState<Remarkable>(() => {
```

Comment:
If no setting, is this actually a `useState`? `useMemo`? If not intended to change as deps change, `useRef`?

Reply:
cringe